pkg/isb/jetstream/writer_test.go

```diff
@@ -140,6 +140,8 @@ func TestForwarderJetStreamBuffer(t *testing.T) {
 
 // TestJetStreamBufferWrite on buffer full
 func TestJetStreamBufferWriterBufferFull(t *testing.T) {
+	// FIXME: #11
+	t.SkipNow()
```

Comment:
It should be working with latest `main`.

Reply:
done